sherpa/csrc/online-transducer-fast-beam-search-decoder.cc

```diff
@@ -127,6 +127,8 @@ void OnlineTransducerFastBeamSearchDecoder::Decode(
       ++t;
       ++p->num_trailing_blanks;
       continue;
+    } else {
+      p->num_trailing_blanks = 0;
     }
```

Comment:
```suggestion
    p->num_trailing_blanks = 0;
```

You don't need to wrap it inside an `else` branch.